src/resources/js/tickets-registration-page.js

```diff
@@ -92,27 +104,94 @@ tribe.tickets.registration = {};
 	};
 
 	/**
-	 * Handle save attendees info form submission.
-	 * Display a message if there are required fields missing.
+	 * Update container status to complete
 	 *
-	 * @since 4.9
+	 * @since TBD
 	 *
 	 * @return void
-	*/
-	obj.handleSaveSubmission = function( e ) {
-		var $form   = $( this );
-		var $fields = $form.parent( obj.selector.fields );
+	 */
+	obj.updateStatusToComplete = function( $event ) {
+		$event.find( obj.selector.status ).removeClass( 'incomplete' );
+		$event.find( obj.selector.status ).find( 'i' ).removeClass( 'dashicons-edit' );
+		$event.find( obj.selector.status ).find( 'i' ).addClass( 'dashicons-yes' );
+	};
+
+	/**
+	 * Update container status to incomplete
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	obj.updateStatusToIncomplete = function( $event ) {
+		$event.find( obj.selector.status ).addClass( 'incomplete' );
+		$event.find( obj.selector.status ).find( 'i' ).addClass( 'dashicons-edit' );
+		$event.find( obj.selector.status ).find( 'i' ).removeClass( 'dashicons-yes' );
+	};
+
+	obj.handleTppSaveSubmission = function( e ) {
+		var $form = $( this );
+		var $fields = $form.closest( obj.selector.fields );
+
+		// hide all messages
+		$fields.find( obj.selector.fieldsErrorRequired ).hide();
 
 		if ( ! obj.validateEventAttendees( $form ) ) {
 			e.preventDefault();
+			$fields.find( obj.selector.fieldsErrorRequired ).show();
+		}
+	};
 
-			$fields.find( obj.selector.fieldsError ).show();
+	/**
+	 * Handle save attendees info form submission via ajax.
+	 * Display a message if there are required fields missing.
+	 *
+	 * @since 4.9
+	 *
+	 * @return void
+	*/
+	obj.handleSaveSubmission = function( e ) {
+		e.preventDefault();
+		var $form = $( this );
+		var $fields = $form.closest( obj.selector.fields );
+		var $event = $fields.closest( obj.selector.container );
 
-			$( 'html, body').animate( {
-				scrollTop: $fields.offset().top
-			}, 300 );
+		// hide all messages
+		$fields.find( obj.selector.fieldsErrorRequired ).hide();
+		$fields.find( obj.selector.fieldsErrorAjax ).hide();
+		$fields.find( obj.selector.fieldsSuccess ).hide();
 
-			return;
+		if ( ! obj.validateEventAttendees( $form ) ) {
+			$fields.find( obj.selector.fieldsErrorRequired ).show();
+			obj.updateStatusToIncomplete( $event )
+		} else {
+			$fields.find( obj.selector.loader ).show();
+
+			var eventId = $event.data( 'event-id' );
+			var params = $form.serializeArray();
+			params.push( { name: 'event_id', value: eventId } );
+			params.push( { name: 'action', value: 'tribe-tickets-save-attendee-info' } );
+			params.push( { name: 'nonce', value: TribeTicketsPlus.save_attendee_info_nonce } );
+
+			$.post(
+				TribeTicketsPlus.ajaxurl,
```

Comment:
@paulmskim This references ET+ code, is this properly checking the var is available for use before trying to run this?

Reply:
Same note as above